.gitignore

```diff
@@ -10,6 +10,6 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
-
+app.json
```

Comment:
Good thought with the API keys. We'll have to make sure everybody knows to recreate app.json file locally.